test/e2e_federation/federated-service.go

```diff
@@ -144,6 +155,7 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 				By(fmt.Sprintf("Verified that services were not deleted from underlying clusters"))
 			})
 		})
+
```

Comment:
Sorry for my wording. if you look at Describe clause carefully, you would find delete test cases inside 'Service creation' clause.